etcdmain/help.go

```diff
@@ -43,6 +45,10 @@ member flags:
 		list of URLs to listen on for peer traffic.
 	--listen-client-urls 'http://localhost:2379,http://localhost:4001'
 		list of URLs to listen on for client traffic.
+        --max-snapshots '` + strconv.Itoa(defaultMaxSnapshots) + `'
+                Maximum number of snapshot files to retain (0 is unlimited).
```

Comment:
Can you change `Maximum` to `maximum` to make it consistent? I prefer `Maximum` but lower-case seems like the convention in this help.